requirements/requirements-ci.txt

```diff
@@ -123,27 +120,27 @@ pycryptodome==3.8.2
 pyflakes==2.1.1
 pygments==2.4.2
 pyhamcrest==1.9.0
-pyinstaller==3.4
+pyinstaller==3.5
```

Comment:
why change pyinstaller version in this PR?

Reply:
I didn't know `requirements/dep upgrade` can take a package name.  Will fix.